gif_frame_num_1: 16
gif_frame_num_2: 31
gif_frame_num_3: 2
gif_frame_num_4: 32
gif_frame_num_5: 32
gif_frame_num_6: 35
gif_frame_num_7: 17

gif_frame_time_1: 0.1
gif_frame_time_2: 0.05
gif_frame_time_3: 0.4
gif_frame_time_4: 0.1
gif_frame_time_5: 0.04
gif_frame_time_6: 0.05
gif_frame_time_7: 0.15
